src/sentry/static/sentry/app/views/eventsV2/savedQuery/index.tsx

```diff
@@ -286,9 +290,57 @@ class SavedQueryButtonGroup extends React.PureComponent<Props, State> {
     );
   }
 
+  renderButtonCreateAlert() {
+    const {eventView, organization, projects} = this.props;
+    // Must have exactly one project selected
+    const hasProjectError = eventView.project.length !== 1;
```

Comment:
What about the -1 project? Which is 'all projects'.

Reply:
@markstory When no project is selected i'm seeing the array as empty, is there some place where the projects array gets a -1 inserted?

Reply:
I figured out where -1 comes from and blocked it as well.